docs/tutorials/optimize/memory/memory_intro.md

```diff
@@ -1,56 +1,5 @@
-## Memory Optimization
+## Memory optimizations
```

Comment:
This page is now very short and doesn't seem to add a lot of content.

Reply:
This is going to be expanded, it just wasn't in my jira ticket